charts/kyverno/values.yaml

```diff
@@ -931,7 +931,10 @@ cleanupJobs:
 admissionController:
 
   # -- Overrides features defined at the root level
-  featuresOverride: {}
+  featuresOverride:
+    admissionReports:
+      # -- Max number of admission reports allowed in flight until the admission controller stops creating new ones
+      backPressureThreshold: 10000
```

Comment:
I suggest we set a smaller threshold, say 1k.

From the reported issues previously, the cluster fell into a bad shape before the cleanup cronjob kicked in (same threshold 10000), 10000 seems too late for the circuit breaker.

Reply:
changed